src/Telnyx.net/LICENSE

```diff
@@ -1,7 +1,6 @@
 The MIT License
 
-Copyright (c) 2019- Telnyx (https://telnyx.com)
-Copyright (c) 2010-2019 Stripe (https://stripe.com)
+Copyright (c) 2019 Telnyx (https://telnyx.com)
```

Comment:
Please revert this to include the Stripe line @rajjy1

Reply:
Done